src/ape/api/transactions.py

```diff
@@ -238,7 +238,7 @@ def raise_for_status(self):
         :class:`~api.providers.TransactionStatusEnum`.
         """
 
-    def await_confirmations(self) -> "ReceiptAPI":
+    def await_confirmations(self, confirmations: Optional[int] = None) -> "ReceiptAPI":
```

Comment:
Maybe calling it `wait_for_confirmation(...) ` would be nicer?

Reply:
I'm down. Maybe it makes sense to make it it's own func? 

It is a little awkward right now... You'll get two progress bars if you do something like this:

```
receipt = <some_tx_here>

receipt.await_confirmations(5)
```

Cuz it calls this func the first time you call the tx, and then you call it again with `receipt.await_confirmations(5)`

Reply:
Are you saying to have two functions? Or refactor this functions name.

Also... I like the name `wait` better, since it's less words. Maybe have that be an optional name for the func?

Reply:
yeah, I think it's awkward right now because it needs to be refactored into the original not waiting on confirmations (no progress bar) and the `wait_for_confirmation(num_confirmations=default)` function displaying it instead

Reply:
Ahhh yeah that makes sense. I might not have bandwidth to make these fixes :/